### PR TITLE
Association

### PR DESCRIPTION
**Pull request by misheck12**
in this milestone i: +1: 

- [ ] Run a program using the command line.

- [ ] Set up associations between classes and objects.

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_accessor :name, :age
```
Comment (rotimiazeez) [inline person.rb]: - [x] Kindly make sure the `Person` has many `rentals`
For example, You should have:
```
attr_accessor :name, :age, :rentals
   
  def initialize(age, name = "Unknown", parent_permission = true)
    .
    .    @rentals = []
  end

  *A method for adding rentals should be present. 
  def add_rental(book, date)
    *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```

Comment (rotimiazeez) [reply person.rb]: - [x] Kindly re-check the project [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_associations.md) and add the required classes and their associations.
For example:
- Create a class Classroom with the following:
   - @label instance variable, should be initialized in the constructor.
   - Setter and getter for @label (remember about attr_accessor).
- Create the has-many/belongs-to relationship between Classroom and Student. The following should be implemented:
   - Create the has-many side (a classroom has many students).
   - Create the belongs-to side (a student belongs to a classroom).
   - Make sure that when adding a student to a classroom it also sets the classroom for the student.
   - Make sure that when setting the classroom for a student it also adds it to the classrooms' students.
- Create a class Book with the following:
   - @title and @author instance variables, should be initialized in the constructor.
   - Setters and getters for instance variables (remember about attr_accessor).

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @misheck12 👏🏽, 
## Good job so far! 👍🏽
### Highlights
 - GitFlow flow best practices have been followed
 - There are no linter errors
 - You have a descriptive PR

#### There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 1 above)
Comment (misheck12) [reply person.rb]: thank you this has been implimented

**Review by t-yanick — APPROVED** [review]
Hi @misheck12, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**